### DISCUSSION TITLE
Feature Request - Bundler - Renovate should skip gracefully local gems

### DISCUSSION THREAD

**Discussion by Djiit** [renovatebot/renovate#32749] [Suggest an Idea]
### Tell us more.

Hello team,

Considering this line:

```
gem 'my-gem', path: 'gems/my-gem'
```
(that is, a gem hosted in the current repository, in a `gems/` subfolder)

Currently, Renovate skip because of `invalid-value`.
I believe it should skip it with reason `internal-package`.

While this is not blocking anything, it would help be more specific about reasons to skip such a gem.

I'm ok to start a PR.

**Comment by rarkins** [comment]
Sounds good
